zulipterminal/core.py

```diff
@@ -312,9 +312,9 @@ def show_about(self) -> None:
             "area:help",
         )
 
-    def show_user_info(self, user_id: int) -> None:
+    def show_user_info(self, user_id: int, cmd: str = "USER_INFO") -> None:
```

Comment:
This doesn't need to be a default value, and would be clearer if it were not.

There's also no need to abbreviate the text to 'cmd' - presumably this is command?

This is a refactor commit - it doesn't change behavior - so it's good to note it as such as an additional commit title area.